src/coreclr/src/debug/daccess/request.cpp

```diff
@@ -701,7 +701,7 @@ ClrDataAccess::GetThreadAllocData(CLRDATA_ADDRESS addr, struct DacpAllocData *da
     Thread* thread = PTR_Thread(TO_TADDR(addr));
 
     data->allocBytes = TO_CDADDR(thread->m_alloc_context.alloc_bytes);
-    data->allocBytesLoh = TO_CDADDR(thread->m_alloc_context.alloc_bytes_loh);
+    data->allocBytesLoh = TO_CDADDR(thread->m_alloc_context.alloc_bytes_uoh);
```

Comment:
do we want to rename this allocBytesLoh field? this GetThreadAllocData is only used by sos I think (not by clrmd?)? @leculver @mikem8361 could you please confirm? for context we are renaming the alloc_bytes_loh field on the alloc_context to alloc_bytes_uoh because we are introducing a concept of uoh which includes loh and additional generations.

Reply:
It looks like SOS doesn't use this API. Not sure about CLRMD.

Reply:
I am suggesting changing this only when clients can use it. No benefit from changing this now, for sure, since there is still only loh in this change, but we may break something. 

---
In reply to: [368189692](https://github.com/dotnet/runtime/pull/1688#discussion_r368189692) [](ancestors = 368189692)

Reply:
We already have tracking bug to revisit this - [https://github.com/dotnet/coreclr/issues/27692](https://github.com/dotnet/coreclr/issues/27692)

---
In reply to: [368208096](https://github.com/dotnet/runtime/pull/1688#discussion_r368208096) [](ancestors = 368208096,368189692)

Reply:
For ClrMD we reimplement all of this interface in C#.  Changing names won't affect us at all, but we require these structures do not change in layout or meaning for backwards compat.

When real changes go in that affect the underlying shape of the GC heap and/or requires changing the shape or usage of DAC data structures it's best to implement a new interface (ISOSDacX or similar) that we can query for to tell us there's a new style of heap involved.